File: src/main/java/life/magpies/registry/GuiditeMaterial.java
package life.magpies.registry;

import net.minecraft.item.ToolMaterial;
import net.minecraft.recipe.Ingredient;

public class GuiditeMaterial implements ToolMaterial {
    @Override
    public int getDurability() {
        return 2200;
    }

    @Override
    public float getMiningSpeedMultiplier() {
        return 5.0F;
    }

    @Override
    public float getAttackDamage() {
        return 3.5F;
    }

    @Override
    public int getMiningLevel() {
        return 1;
    }

    @Override
    public int getEnchantability() {
        return 22;
    }

    @Override
    public Ingredient getRepairIngredient() {
        return Ingredient.ofItems(ModItems.CITRINE);
    }

    public static final GuiditeMaterial INSTANCE = new GuiditeMaterial();
}

File: src/main/kotlin/life/magpies/registry/items/GuiditeMaterial.kt
package life.magpies.registry.items

import life.magpies.registry.ModItems
import net.minecraft.item.ToolMaterial
import net.minecraft.recipe.Ingredient


object CitrineMaterial:ToolMaterial {
    override fun getDurability(): Int {
        return 2200
    }

    override fun getMiningSpeedMultiplier(): Float {
        return 5.0f
    }

    override fun getAttackDamage(): Float {
        return 3.5f
    }

    override fun getMiningLevel(): Int {
        return 1
    }

    override fun getEnchantability(): Int {
        return 18
    }

    override fun getRepairIngredient(): Ingredient {
        return Ingredient.ofItems(ModItems.CITRINE)
    }

    val INSTANCE: CitrineMaterial = CitrineMaterial

}